### DISCUSSION TITLE
How can I use Docker to run openroad flow on the host machine?

### DISCUSSION THREAD

**Discussion by huang-kaixin** [The-OpenROAD-Project/OpenROAD-flow-scripts#1769] [Q&A]
I am not very familiar with Docker building OpenRoad, and I run it based on [./build_openroad.sh ](https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildWithDocker.html) has failed. Is there an image that can be directly pulled from Dockerhub and run Openroad flow on my host? Could you please provide all the building processes? Thank you for your reply!

**Comment by maliberty** [comment]
@vvbandeira is such an image available yet?  We do plan to produce one.
